File: Inline Method.py
// After

class PizzaDelivery:
    #...
    def getRating(self):
        return 2 if self.moreThanFiveLateDeliveries() else 1
  
    def moreThanFiveLateDeliveries(self):
        return self.numberOfLateDeliveries > 5

//Refactoring

class PizzaDelivery:
  #...
  def getRating(self):
    return 2 if self.numberOfLateDeliveries > 5 else 1

File: Inline_Method.js
// After
class PizzaDelivery {
  function getRating(self) {
    return self.moreThanFiveLateDeliveries() ? 2 : 1;
  }    

  function moreThanFiveLateDeliveries(self) {
      return self.numberOfLateDeliveries > 5;
  }
}    

// Refactoring
class PizzaDelivery {
  function getRating(self) {
    return self.numberOfLateDeliveries > 5 ? 2 : 1;
  }  
}  
